<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="9" hostname="DESKTOP-D9BEU1J" failures="1" tests="3" name="Test1" time="23.354" errors="0" timestamp="2022-02-10T21:28:01 EET">
  <testcase classname="Tests.UMM1" name="UMM1PS1" time="2.884">
    <failure type="java.lang.AssertionError" message="expected [This user already exist.] but found [Sign up successful.]">
      <![CDATA[java.lang.AssertionError: expected [This user already exist.] but found [Sign up successful.]
at Tests.UMM1.UMM1PS1(UMM1.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- UMM1PS1 -->
  <testcase classname="Tests.UMM2" name="UMM2PS1" time="6.391"/>
  <testcase classname="Tests.AM6" name="AM6PS1" time="7.154"/>
  <testcase classname="Tests.UMM1" name="UMM1AS3" time="0.0">
    <ignored/>
  </testcase> <!-- UMM1AS3 -->
  <testcase classname="Tests.UMM1" name="UMM1AS1" time="0.0">
    <ignored/>
  </testcase> <!-- UMM1AS1 -->
  <testcase classname="Tests.AM6" name="AM6AS1" time="0.0">
    <ignored/>
  </testcase> <!-- AM6AS1 -->
  <testcase classname="Tests.UMM2" name="UMM2AS2" time="0.0">
    <ignored/>
  </testcase> <!-- UMM2AS2 -->
  <testcase classname="Tests.UMM2" name="UMM2AS3" time="0.0">
    <ignored/>
  </testcase> <!-- UMM2AS3 -->
  <testcase classname="Tests.UMM1" name="UMM1AS2" time="0.0">
    <ignored/>
  </testcase> <!-- UMM1AS2 -->
  <testcase classname="Tests.UMM2" name="UMM2AS1" time="0.0">
    <ignored/>
  </testcase> <!-- UMM2AS1 -->
  <testcase classname="Tests.TestParameters" name="@AfterTest TearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest TearDown -->
  <testcase classname="Tests.TestParameters" name="@AfterTest TearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest TearDown -->
</testsuite> <!-- Test1 -->
